src/fpm_compiler.F90

```diff
@@ -539,6 +548,67 @@ function get_module_flag(self, path) result(flags)
 end function get_module_flag
 
 
+function get_feature_flag(self, feature) result(flags)
+    class(compiler_t), intent(in) :: self
+    character(len=*), intent(in) :: feature
+    character(len=:), allocatable :: flags
+
+    flags = ""
+    select case(feature)
+    case("no-implicit-typing")
+       select case(self%id)
+       case(id_caf, id_gcc, id_f95)
+           flags = flag_gnu_no_implicit_typing
+
+       end select
+
+    case("implicit-typing")
+       select case(self%id)
+       case(id_lfortran)
+           flags = flag_lfortran_implicit_typing
+
+       end select
+
+    case("no-implicit-external")
+       select case(self%id)
+       case(id_caf, id_gcc, id_f95)
+           flags = flag_gnu_no_implicit_external
+
+       end select
+
+    case("implicit-external")
+       select case(self%id)
+       case(id_lfortran)
+           flags = flag_lfortran_implicit_external
+
+       end select
+
+    case("free-format")
+       select case(self%id)
+       case(id_caf, id_gcc, id_f95)
+           flags = flag_gnu_free_format
+
+       end select
+
+    case("fixed-format")
+       select case(self%id)
+       case(id_caf, id_gcc, id_f95)
+           flags = flag_gnu_fixed_format
+
```

Comment:
Adds fixed-form compile flags (also fixes indentation on the fist couple of lines)

```suggestion
        select case(self%id)
        case(id_caf, id_gcc, id_f95, id_flang_new)
           flags = flag_gnu_fixed_format

        case(id_intel_classic_nix, id_intel_llvm_nix, id_intel_classic_mac)
            flags = flag_intel_fixed_format

        case(id_intel_classic_windows, id_intel_llvm_windows)
            flags = flag_intel_fixed_format_win

        case(id_pgi, id_nvhpc, id_flang, id_f18)
            flags = flag_pgi_fixed_format

        case(id_nag)
            flags = flag_nag_fixed_format

```